libkbfs/conflict_resolver.go

```diff
@@ -374,7 +374,7 @@ func fileWithConflictingWrite(unmergedChains, mergedChains *crChains,
 // unmerged paths that need to be checked for conflicts later in
 // conflict resolution, for all subdirectories of the given path.
 func (cr *ConflictResolver) checkPathForMerge(ctx context.Context,
-	unmergedChain *crChain, unmergedPath path, unmergedChains *crChains,
+	unmergedChain *crChain, unmergedPath path, unmergedChains,
```

Comment:
Done, moved to next line (and everywhere else). Add more lines, but I think it's easier to read anyway.